lib/syslog.rb

```diff
@@ -1,10 +1,14 @@
 begin
-  require 'syslog_ext'
+  if RUBY_ENGINE == 'jruby'
+    require 'syslog/jruby'
+  else
+    require 'syslog_ext'
+  end
 rescue LoadError
   raise LoadError.new(<<-EOS)
-    Can't load Syslog!
-
-    Syslog is not supported on your system. For Windows
-    we recommend using the win32-eventlog gem.
-  EOS
-end
+      Can't load Syslog!
+  
+      Syslog is not supported on your system. For Windows
+      we recommend using the win32-eventlog gem.
+    EOS
+end
```

Comment:
This may be a trivial issue, but perhaps ensure a line ending here?

Reply:
Added the newline and re-pushed.